audonnx/core/testing.py

```diff
@@ -0,0 +1,161 @@
+import typing
+
+import numpy as np
+import onnx
+
+import audeer
+import audobject
+import audonnx
+
+
+def create_model(
+        root: str,
+        shapes: typing.Sequence[typing.Sequence[int]],
+        *,
+        value: float = 0.,
+        dtype: int = onnx.TensorProto.FLOAT,
+        opset_version: int = 14,
+) -> audonnx.Model:
+    r"""Create test model.
+
+    Creates a model that outputs
+    arrays filled with ``value``
+    of the given ``shapes``.
+    The model is stored under ``root``
+    and can be loaded with
+    :func:`audonnx.load`.
+
+    Args:
+        root: folder where model is stored
+        shapes: list with model shapes
```

Comment:
This needs a little bit more documentation on how the shapes should be given, e.g. `[[output0-dim0, output0-dim1], [output1-dim0]]`. Or maybe: ", e.g. for a model with two output nodes ``[[3], [1, None, 2]]``".

It also seems to affect input and output in the way at the moment. Is this always desirable?

Reply:
Maybe it would also make sense to allow providing names for the inputs/outputs?

E.g. `outputs={'output1': [1, -1, 1]}`

Reply:
On the other hand, the current naming scheme is consistent and predictable, which might also be nice for testing.

Reply:
How about:

![image](https://user-images.githubusercontent.com/10383417/172608101-660e795d-ae7c-4755-b827-15a50c11bb20.png)

> It also seems to affect input and output in the way at the moment. Is this always desirable?

The problem are the dynamic axis, as for those we cannot set a fixed shape in the model graph (otherwise we could simply create completely unconnected input and output nodes). But since dynamic dimensions were the main motivation to create a testing module, I decided to create an identity network. I think for our purposes this should be fine. Or do you have a better idea?

Reply:
> Maybe it would also make sense to allow providing names for the inputs/outputs?
> 
> E.g. outputs={'output1': [1, -1, 1]}

But then we should probably also provide an option to provide names for the inputs. But then it gets complicated again :)

Reply:
OK, then let's stick with this solution until we really need inputs that are different from the outputs.
But I would also mention it in the docs, e.g.

"list with shapes defining the output nodes of the model. The model will have the same number of input nodes, copying the shapes from the output nodes"

Reply:
Ok, I updated the docstring accordingly, see image in description.